scripts/bootstrap.sh

```diff
@@ -456,26 +484,32 @@ update_devstack() {
 # START CORE EXECUTION
 ###############################################
 
-# source our utilities for logging and other base functions (we need this staged with the installer script)
-# the file needs to be first downloaded from the public repository
+# get current dir
 CURRENT_PATH="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-UTILITIES_PATH=$CURRENT_PATH/../templates/stamp/utilities.sh
 
-# check if the utilities file exists. If not, bail out.
-if [[ ! -e $UTILITIES_PATH ]]; 
-then  
-    echo :"Utilities not present"
-    exit 3
+# pass existing command line arguments
+parse_args "$@"
```

Comment:
we now parse arguments BEFORE sourcing utilities. This let's us know which branch to download the utilities file from if it's missing